src/react-components/src/components/01-atoms/Forms/FormDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+// AF-121 Form Dropdown
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Icon from "../Images/Icons/Icon";
+
+export interface FormDropdownProps {
+  dropdownId: string;
+  blockName?: string;
+  labelText?: string;
+  ariaLabel?: string;
+  disabled?: boolean;
+  options: string[];
+  selectedOption?: string;
+  onSelectBlur: (event: React.FormEvent) => void;
+  onSelectChange: (event: React.FormEvent) => void;
+}
+
+export default function FormDropdown(props: FormDropdownProps) {
+  const { dropdownId, blockName, labelText, options, ariaLabel, disabled, selectedOption, onSelectBlur, onSelectChange } = props;
+  if (!labelText) {
+    if (!ariaLabel) {
+      // TODO: Assign aria-labeledBy to labelText
+      throw new Error("Either labelText or ariaLabel must be defined");
+    }
+  }
+
+  let formItemBlockName = blockName ? blockName : "form-item";
+  let labelClassName = bem("label", ["textField"], formItemBlockName);
+
+  return <div className={bem("dropdown", [], formItemBlockName)}>
+    {labelText && (<label htmlFor={dropdownId} className={labelClassName}>{labelText}</label>)}
+
+    <select id={dropdownId}
+    className={bem("select", [], formItemBlockName)}
+    value={selectedOption}
+    onChange={onSelectChange}
+    onBlur={onSelectBlur}
+    aria-label={ariaLabel}
+    disabled={disabled ? disabled : false}>
+        { options.map((child, key) => {
+          return <option key={key.toString()} aria-selected={child === selectedOption} value={child}>{ child }</option>;
+        }) }
+    </select>
+
+    <Icon decorative={true} name={'arrow'} modifiers={['small']} />
+  </div>
```

Comment:
I'm going to be unnecessarily nitpicky:
I think it'll be more readable if it was formatted like below but definitely not needed! Just a preference and if you want to introduce `[prettier](https://prettier.io/)`, it'll format it similarily. You are missing a comma at the end of the `return` (the linter didn't pick it up?).
```
return (
    <div className={bem("dropdown", [], formItemBlockName)}>
      {labelText && (<label htmlFor={dropdownId} className={labelClassName}>{labelText}</label>)}

      <select id={dropdownId}
        className={bem("select", [], formItemBlockName)}
        value={selectedOption}
        onChange={onSelectChange}
        onBlur={onSelectBlur}
        aria-label={ariaLabel}
        disabled={disabled ? disabled : false}
      >
          { options.map((child, key) => {
            return <option key={key.toString()} aria-selected={child === selectedOption} value={child}>{ child }</option>;
          }) }
      </select>

      <Icon decorative={true} name={'arrow'} modifiers={['small']} />
    </div>
);
```